File: app/src/main/java/com/roy93group/cpuinfo/widgets/swiperv/listener/SwipeFractionListener.java
package com.roy93group.cpuinfo.widgets.swiperv.listener;

import com.roy93group.cpuinfo.widgets.swiperv.SwipeMenuLayout;


public interface SwipeFractionListener {
    void beginMenuSwipeFraction(SwipeMenuLayout swipeMenuLayout, float fraction);

    void endMenuSwipeFraction(SwipeMenuLayout swipeMenuLayout, float fraction);
}

File: app/src/main/java/com/roy93group/cpuinfo/widgets/swiperv/listener/SwipeFractionListener.kt
package com.roy93group.cpuinfo.widgets.swiperv.listener

import com.roy93group.cpuinfo.widgets.swiperv.SwipeMenuLayout

interface SwipeFractionListener {
    fun beginMenuSwipeFraction(swipeMenuLayout: SwipeMenuLayout?, fraction: Float)
    fun endMenuSwipeFraction(swipeMenuLayout: SwipeMenuLayout?, fraction: Float)
}
